<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extraPolacion</class>
 <widget class="QDialog" name="extraPolacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extrapolaci√≥n de avance</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>509</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 134, 136);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="mainGB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>901</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 231, 231);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>381</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Predecir siguiente punto en ruta</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="lbSensor">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sensor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="5">
      <widget class="QComboBox" name="cboSensor">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLabel" name="lbIdTortuga">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Id Tortuga</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="4">
      <widget class="QLineEdit" name="lneIdTortuga">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="7">
      <widget class="QLabel" name="lbNombreTortuga">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre tortuga</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLineEdit" name="lneNombreTortuga">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QLabel" name="lbSubespecie">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Subespecie</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5" colspan="3">
      <widget class="QLineEdit" name="lneSubespecie">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="6">
      <widget class="QLabel" name="lbInvestigador">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Investigador</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6" colspan="2">
      <widget class="QLineEdit" name="lneInvestigador">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbZona">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zona</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="7">
      <widget class="QLineEdit" name="lneZona">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="lbMunicipio">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Municipio</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="5">
      <widget class="QLineEdit" name="lneMunicipio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="lbEstado">
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="6">
      <widget class="QLineEdit" name="lneEstado">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="showMapPB">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton#showMapPB {
background-color: rgb(120, 190, 151);
 }

 QPushButton#showMapPB:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
    </property>
    <property name="text">
     <string>Mostrar mapa</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/Images/381031-200.ico</normaloff>:/Images/381031-200.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPB">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton#exitPB {
background-color: rgb(120, 190, 151);
 }

 QPushButton#exitPB:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>46</width>
      <height>46</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitPB</sender>
   <signal>clicked()</signal>
   <receiver>extraPolacion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
